### ISSUE TITLE
Message "Failed to extract server files" when trying to extract server.zip in Cuberite 1.6.0

### ISSUE THREAD

**Issue by ElectronicallyE** [cuberite/android#70]
Hello,

I am using a Samsung Galaxy J1 Mini (SM-J105Y). I have downloaded and installed the Cuberite 1.6.0 APK from F-Droid. When clicking the green circle "Install Cuberite" on the main page, a popup appears stating "Downloading Cuberite" with no progress (0 %). After approximately 30 seconds, I get the message "Failed to extract server files". I've instead downloaded 'armeabi-v7a.zip' (due to the ARMv7 architecture of the J1 Mini) and 'server.zip' directly to the phone from [Jenkins](https://builds.cuberite.org/job/android/lastSuccessfulBuild/artifact/android/Server/).

Under 'Settings' > 'Install Cuberite manually', clicking 'Select Cuberite executable zip file' > 'armeabi-v7a.zip' gives the message "Successfully installed Cuberite!". If I click "Select server.zip file" and select "server.zip", I get the message "Failed to extract server files".

The debug information is:
Running on Android 5.1.1 (API Level 22)
Using ABI armeabi-v7a
IP: 172.20.10.11
Private directory: /data/data/org.cuberite.android/files
Public directory: /storage/emulated/0
Storage location: BLANK
Download URL: https://download.cuberite.org/androidbinaries/

I have 3.8 GB available in storage. I can also confirm I did not download the .sha1 files.

Help would be much appreciated in resolving this issue.

**Comment by ElectronicallyE** [comment]
@mathiascode, thanks for rectifying this so quickly. In order to use the fix, must I compile the code instead of downloading the latest release (1.6.0)?

**Comment by mathiascode** [comment]
Unfortunate regression, but I finally got Android emulators working so I can test older versions properly again. :)

You can test the apk I've attached below to make sure everything is working correctly on your device. Note that it's not compatible with the F-Droid apk, so you have to uninstall the Cuberite app first.

[app-debug.zip](https://github.com/cuberite/android/files/14123622/app-debug.zip)

If everything is fine, I'll tag a 1.6.1 release, which F-Droid will hopefully pick up after a few days.

**Comment by ElectronicallyE** [comment]
Just tried downloading app-debug.zip. It is empty (0 B). Could you reupload it?

**Comment by mathiascode** [comment]
Drag and drop in Linux strikes again... Try this one:

[app-debug.zip](https://github.com/cuberite/android/files/14123728/app-debug.zip)

**Comment by ElectronicallyE** [comment]
It works! Got the message "Successfully installed Cuberite!" when clicking the green circle containing "Install Cuberite". It now says "Start Cuberite".

After clicking "Start Cuberite", the 'Console' page appears with a print out. I've also been able to log into the portal (http://172.20.10.11:8080).

I've also tried connecting to the server, but getting a "Timed Out" error. I'll publish a new issue describing it in more detail.
